src/components/CodeHighlight/LogColorizer.js

```diff
@@ -0,0 +1,194 @@
+import React, {
+  memo,
+  useState,
+  Fragment,
+} from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  ButtonGroup,
+  Button,
+  Select,
+} from '@folio/stripes/components';
+
+import { LOG_COLORIZER } from '../../utils/constants';
+import { CodeHighlight } from './CodeHighlight';
+import * as Languages from './Languages';
```

Comment:
```suggestion
import { LANGUAGES } from './Languages';
```

Reply:
This place is correct, cause we use Languages object itself later in the code.

Reply:
As I can see it is not used in code

Reply:
Done